extensions/wikia/CuratedContent/CuratedContentValidatorController.class.php

```diff
@@ -15,6 +16,11 @@ public function __construct() {
 	}
 
 	public function validateSection() {
+		global $wgRequest;
```

Comment:
Lately I bumped into this https://github.com/Wikia/app/blob/XW-396/skins/oasis/modules/BodyController.class.php#L121, where `global $wgRequest` was exchanged for `F::app()->wg->Request`. Not using globals sounds right, but I'm just a low level PHP warlock. ;)